cpp/include/cudf/dictionary/dictionary_column_view.hpp

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/column/column_view.hpp>
+#include <cudf/column/column.hpp>
+
+namespace cudf
+{
+/**
+ * @brief Given a column-view of DICTIONARY type, an instance of this class
```

Comment:
Say what the class is first, not the inputs.

```suggestion
 * @brief A view of a dictionary column.
 *
 * Given a `column_view` of `DICTIONARY` type, an instance of this class
```